src/sas/qtgui/Perspectives/Fitting/PolydispersityWidget.py

```diff
@@ -478,6 +481,14 @@ def updateFittedValues(row: int) -> None:
             # Nsigs
             param_repr = GuiUtils.formatNumber(param_dict[param_name][5+ioffset], high=True)
             self.poly_model.item(row, 5+joffset).setText(param_repr)
+            # Function
+            param_repr = param_dict[param_name][6+ioffset]
+            self.poly_model.item(row, 6+joffset).setText(param_repr)
```

Comment:
Setting the text on the combobox is not necessary.
Column 6 always contains a combo box (`addNameToPolyModel`)